front/src/applications/stdcm/utils/formatStdcmConfV2.ts

```diff
@@ -246,6 +252,9 @@ export const formatStdcmPayload = (
     margin: createMargin(validConfig.margin),
     rolling_stock_id: validConfig.rollingStockId,
     speed_limit_tags: validConfig.speedLimitByTag,
+    total_mass: validConfig.totalMass ? validConfig.totalMass * 1000 : undefined, // Api require total_mass in kg
+    max_speed: validConfig.maxSpeed ? validConfig.maxSpeed * 1000 : undefined, // The form is in km/s but the Api is in m/s
```

Comment:
The form is (should be, see other comment) in km/h. Use the util `kmhToMs` in `physics.ts`